backend/.gitignore

```diff
@@ -0,0 +1,3 @@
+target/
+dist/
+*.env
```

Comment:
this is covered by the root `.gitignore` no?

Reply:
isn't 2 always better than 1? I removed it and just took static/ off the root .gitignore